packages/astro/src/core/sync/index.ts

```diff
@@ -124,7 +125,9 @@ export async function syncInternal({
 			});
 			await contentLayer.sync();
 			settings.timer.end('Sync content layer');
-		} else if (fs.existsSync(getContentPaths(settings.config, fs).contentDir.href)) {
+		} else if (
+			fs.existsSync(normalizePath(fileURLToPath(getContentPaths(settings.config, fs).contentDir)))
```

Comment:
We shouldn't need `normalizePath` here. That's not what it's for. Shouldn't the `fileUrlToPath` be enough?